examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,385 @@
+"""
+
+Modeling Cavitation
+-------------------
+This example examines the pressure-driven cavitating flow of water
+through a sharp-edged orifice. This is a typical configuration in
+fuel injectors, and brings a challenge to the physics and numerics
+of cavitation models because of the high pressure differentials
+involved and the high ratio of liquid to vapor density.
+
+This example uses the multiphase modeling capability of Ansys
+Fluent, and will be able to predict the strong cavitation near the
+orifice after flow separation at a sharp edge.
+
+**Workflow tasks**
+
+This example demonstrates how to do the following:
+
+- Set boundary conditions for internal flow
+- Use the mixture model with cavitation effects
+- Calculate a solution using the pressure-based coupled solver
+
+**Problem description**
+
+The problem considers the cavitation caused by the flow separation
+after a sharp-edged orifice. The flow is pressure driven, with an
+inlet pressure of 5 x 10^5 Pa and an outlet pressure of 9.5 x 10^4 Pa.
+The orifice diameter is 4 x 10^-3 m, and the geometrical parameters
+of the orifice are D/d = 2.88 and L/d = 4, where D, d, and L are the
+inlet diameter, orifice diameter, and orifice length respectively.
+"""
+# sphinx_gallery_thumbnail_path = '_static\cavitation_model.png'
```

Comment:
```suggestion
```
This sphinx gallery command needs to be in a code block, otherwise it will show up in the rendered webpage (see my other suggestion)

Reply:
Thank you, the changes have been added.